homeassistant/components/mobile_app/helpers.py

```diff
@@ -146,3 +147,15 @@ def webhook_response(data, *, registration: Dict, status: int = 200,
 
     return Response(text=data, status=status, content_type='application/json',
                     headers=headers)
+
+
+async def get_config_entry(hass: HomeAssistantType,
+                           webhook_id: str) -> ConfigEntry:
+    """Return the config entry for the given webhook ID."""
+    device_registry = await dr.async_get_registry(hass)
```

Comment:
This seems inefficient.

We can either store the config entry in `hass.data` during `async_setup_entry`. But if you want to look it up, you can just get the entries for your domain using `entries = hass.config_entries.async_entries(DOMAIN)`. Then go through the entries and find the one with `entry.data[ATTR_WEBHOOK_ID] == `